docs/hub/repositories-pull-requests-discussions.md

```diff
@@ -21,6 +21,39 @@ At a high level, the aim is to build a simpler version of other git hosts' (like
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/discussions-view-dark.png"/>
 </div>
 
+## Editing a disucssion / pull request title
+
+If you opened a PR or discussion, or are the author of the repository, you can edit the discussion title by clicking on the pencil button.
```

Comment:
```suggestion
If you opened a PR or discussion, or are the author of the repository (or otherwise have write access to it), you can edit the discussion title by clicking on the pencil button.
```